### PR TITLE
Feature/more permissive grafana

### PR DESCRIPTION
**Pull request by hez**
A few quality of life Grafana changes

- Killed off docker compose default env params (was causing massive problems with ver of docker compose available on the raspberry pi)
- Changed up the Grafana init so it has a writable Dashboards directory that is mapped back to the host.  Previously the Dashboards directory was in the container and unwritable.

### CODE REVIEW

File: conf/grafana/provisioning/dashboards/all.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- name: 'default'       # name of this dashboard configuration (not dashboard itself)
+  org_id: 1             # id of the org to hold the dashboard
+  folder: ''            # name of the folder to put the dashboard (http://docs.grafana.org/v5.0/reference/dashboard_folders/)
+  type: 'file'          # type of dashboard description (json files)
+  options:
+    folder: '/dashboards'       # where dashboards are
```
Comment (jrstarke) [inline conf/grafana/provisioning/dashboards/all.yaml]: According to the logs, `folder` is deprecated, and `path` should be used instead. I tested replacing `folder:` with `path:` and it works as expected.

**Review by jrstarke — CHANGES_REQUESTED** [review]
Change looks really good. One change.

File: conf/grafana/provisioning/dashboards/all.yaml (hunk 1 above)
Comment (hez) [reply conf/grafana/provisioning/dashboards/all.yaml]: ah! interesting, I'll take a look, some how didn't see that msg :\

Comment (hez) [reply conf/grafana/provisioning/dashboards/all.yaml]: 👍 done

**Review by jrstarke — APPROVED** [review]

**Comment by jrstarke** [comment]
Awesome. Thanks for the change. Looks great 👍

**Pull request merged**